### PR TITLE
Mention that directives are supported in .mdx

### PR DESCRIPTION
**Pull request by calebeby**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

The `.mdx` page already mentions that this is supported, but I figured it was worth mentioning it in the directives page since it currently just says directives are supported in `.astro` files which misled me.

### CODE REVIEW

**Comment by sarah11918** [comment]
Hi @calebeby, thanks for this!  You're right, in order to use a UI framework component in MDX, you do need to use a client directive. And, *some* of the other template directives work, but not *all* of them. So, I'm hesitant to say at the top of the page such a general statement as "these work in MDX."

Was it a client directive on a framework component you ended up using in MDX? Because we could then instead try to mention *that* if not having it there is misleading.

As for the other directives, I think the intention is for them to be used in `.astro` files, and if some happen to work in MDX, that might be a coincidence. So, we also don't want to document something that isn't a planned, supported feature.

Given all that, and your experience using this page while working with Astro and what you were looking for, do you have an idea for a more specfic, true statement we could use here?

**Comment by calebeby** [comment]
Yeah it was a client directive. I can adjust the change to make it more specific.

File: src/pages/en/reference/directives-reference.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Template Directives Reference
 i18nReady: true
 ---
 
-**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files).
+**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files), and in `.mdx` files.
```
Comment (sarah11918) [inline src/pages/en/reference/directives-reference.md:7]:
```suggestion
**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files), and some can also be used in `.mdx` files.
```
Do you think it works if we just say this @calebeby?

Comment (calebeby) [reply src/pages/en/reference/directives-reference.md:7]: Yeah 👍

**Comment by sarah11918** [comment]
Woo! And we're good! Thanks for this helpful first-time contribution, @calebeby! Welcome to Team Docs and the Face Pile!  🥳

**Pull request merged**